packages/blocks/src/__internal__/rich-text/rich-text-operations.ts

```diff
@@ -149,11 +149,13 @@ export function handleIndent(page: Page, model: ExtendedModel, offset = 0) {
     children: [...previousSibling.children, model, ...children],
   });
 
-  // FIXME: after quill onload
   requestAnimationFrame(() => {
```

Comment:
Instead of using `requestAnimationFrame`, can we add an `onLoad` signal for virgo?

Reply:
What `onload` means? Are you referring to the synchronization of the selection area or whether the text status has been loaded into the DOM?

Reply:
> What `onload` means? Are you referring to the synchronization of the selection area or whether the text status has been loaded into the DOM?

I think it means the editor has loaded all the dom it needs and ready for input.